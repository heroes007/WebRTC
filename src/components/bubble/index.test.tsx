import React from 'react';
import { shallow, mount, render } from 'enzyme';
import renderer from 'react-test-renderer';
import Bubble from './index';

describe('bubble test', () => {
  it('render Bubble default', () => {
    const component = renderer.create(<Bubble>H</Bubble>);
    
    expect(component.toJSON()).toMatchSnapshot();
  });

  it('test size prop', () => {
    const component = shallow(<Bubble size="small">H</Bubble>);
    expect(component.hasClass('small')).toBe(true);
  });

  it('test type prop', () => {
    const component = shallow(<Bubble type="ball">H</Bubble>);
    expect(component.hasClass('ball')).toBe(true);
  });
});