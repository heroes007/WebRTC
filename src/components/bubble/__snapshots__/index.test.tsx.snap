// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bubble test render Bubble default 1`] = `
<span
  className="circle-icon-text middle"
>
  H
</span>
`;
